src/server/lib/example.js

```diff
@@ -6,3 +6,21 @@
   return `${count} ${plural}`;
 };
 
+
+exports.pluralize_v2 = function (count, singular, plural) {
+  if (count === 1) {
+    return `${count} ${singular}`;
+  }
+  plural = plural || `${singular}s`;
+  return `${count} ${plural}`;
+};
+
+exports.method = function (count) { 
+  for(let i = 0; i < count; i++){
+    const j = 9
+    while(j>count){
```

Comment:
Codacy has a fix for the issue: [Replace `(j>count)` with `·(j·>·count)·`](http://sh.se.codacy.org/gh/codacy-acme/nodeproject/pullRequest?prid=24)
```suggestion
    while (j > count) {
```